core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/cache/NamedViewCache.java

```diff
@@ -253,36 +211,46 @@ private void add(final NamedViewDetail namedViewDetail, final boolean overwrite,
 
         try {
             existing = getFromCache(namedViewDetail.getName());
-        } catch (final CacheOperationFailedException e) { // if there is no existing NamedView add one
+        } catch (final
```

Comment:
Can this be moved back onto one line to make it easier to read

Reply:
yuk, don't know why that happened.